### DISCUSSION TITLE
Issues with HW03 Images

### DISCUSSION THREAD

**Discussion by kguert** [cooperrc/computational-mechanics#200] [Q&A]
Hey Everyone, 

I've been having issues getting the images from the video to load for the work to Part 1 of HW03. Does anyone else have the same issue or ideas to resolve?

![ME3255](https://github.com/cooperrc/computational-mechanics/assets/135081793/ae35b509-6526-4311-b530-60b2d8b28a70)

**Comment by cooperrc** [comment]
It looks like you might need to run cells from top-to-bottom. 

Try restarting the kernel (Click kernel, restart), then run the notebook from starting from the first cell.

**Reply by kguert** [reply]
I was trying that last night with no luck. I'll try it again tonight and see how it goes.

**Reply by cooperrc** [reply]
I'm giving it another shot too. I'll post a video if there's any tips/tricks I find

**Reply by cooperrc** [reply]
[Creating, saving, and using data from videos](https://youtu.be/9cmOelJzWXY)

^ this was my process for getting the coordinates from the videos for HW_03. 

as a minimum, I think you would need to run these commands before trying to view video frames, 

```python
%matplotlib notebook
```

```python
import imageio 
import numpy as np
import matplotlib.pyplot as plt

plt.style.use('fivethirtyeight')

from ipywidgets import widgets

from urllib.request import urlretrieve
URL = 'http://go.gwu.edu/engcomp3vid1?accessType=DOWNLOAD'
urlretrieve(URL, 'Projectile_Motion.mp4')
```

**Reply by kguert** [reply]
I tried it again today and it worked fine. Maybe because I accessed it on a different computer while the window was still open on my home computer? I'm not sure.
